libcontainer/rootfs_linux.go

```diff
@@ -158,6 +180,19 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 				return err
 			}
 		}
+		if copyUp {
+			f, err := os.Open(tmpFilePath)
```

Comment:
If you combine this with the note above about the `else`, it looks like this will cause `copyup` to always fail if the target mount point doesn't exist. Is that intentional?